contracts/base/Lenders.sol

```diff
@@ -155,6 +129,32 @@ contract Lenders is Base, LendersInterface {
         return amount;
     }
 
+    /**
+        @notice It initializes this contract instance.
+        @param zTokenAddress zToken contract address.
+        @param lendingPoolAddress lending pool contract address.
+        @param interestConsensusAddress interest consensus contract address.
+        @param settingAddress settings contract address.
+        @param marketsAddress markets contract address.
+     */
+    function initialize(
+        address zTokenAddress,
+        address lendingPoolAddress,
+        address interestConsensusAddress,
+        address settingAddress,
+        address marketsAddress
+    ) external isNotInitialized() {
+        zTokenAddress.requireNotEmpty("ZTOKEN_MUST_BE_PROVIDED");
+        lendingPoolAddress.requireNotEmpty("LENDING_POOL_MUST_BE_PROVIDED");
+        interestConsensusAddress.requireNotEmpty("CONSENSUS_MUST_BE_PROVIDED");
+
+        _initialize(settingAddress, marketsAddress);
+
+        zToken = zTokenAddress;
+        lendingPool = lendingPoolAddress;
+        interestConsensus = InterestConsensusInterface(interestConsensusAddress);
+    }
+
```

Comment:
Any reason this was moved down here? Makes more sense to keep it towards the top.

Reply:
Yes, this is due to gas consumption reasons. The functions that will be invoked more frequently should be on the top. So they consume less gas when EVM compares the sig function.